test/bindings.test.js

```diff
@@ -33,15 +33,15 @@ const goodOptions = {
   // oneFilePerEvent: 1,             // not supported by the node agent
   ec2MetadataTimeout: 3000,
   proxy: 'http://proxy-host:10101',
-  mode: 0
+  metricFormat: 1
 }
 
 describe('bindings.oboeInit()', function () {
   // Important: when only services key is provided the host is set to production by default
   // tests here always run against production collector. no way to "override" with settings.
 
   it('should initialize oboe with only a service key', function () {
-    const result = bindings.oboeInit({ serviceKey: `${env.SW_TEST_PROD_SERVICE_KEY}`, mode: 1 })
+    const result = bindings.oboeInit({ serviceKey: `${env.SW_TEST_PROD_SERVICE_KEY}`, metricFormat: 0 })
```

Comment:
Curious if `metricsFormat` needs to be passed in, which goes against the (inherited) contradiction with test description -- from quick look at the bindings.cc code it _should_ be set to the default 0 in the call to `oboe_init_options_set_defaults`.

Reply:
Just tested and it does, updating the test to reflect that and catch it if it doesn't anymore